src/common/greptimedb-telemetry/src/lib.rs

```diff
@@ -259,7 +261,9 @@ impl GreptimeDBTelemetry {
                 };
 
                 if let Some(client) = self.client.as_ref() {
-                    info!("reporting greptimedb version: {:?}", data);
```

Comment:
the `self` is mutable , looks like we don't need an `AtomicUsize`. Just use a `usize` type instead.